brightener.h

```diff
@@ -9,9 +9,10 @@ struct Image {
 
 class ImageBrightener {
 private:
-	Image m_inputImage;
+	const std::unique_ptr <Image>& m_inputImage;
 public:
-	ImageBrightener(Image& inputImage);
+	ImageBrightener(const std::unique_ptr<Image>& inputImage);
+	//ImageBrightener(Image& inputImage);
```

Comment:
Do not leave commented code in commits. If we need older code, it will be present in the GitHub history